boards/shields/link_board_eth/doc/index.rst

```diff
@@ -0,0 +1,119 @@
+.. _link_board_eth_shield:
+
+PHYTEC link board ETH
+#####################
+
+Overview
+********
+
+The link board ETH is an ethernet extension board compatible with the Arduino
+UNO R3 connector. It is based on the ENC424J600 ethernet controller.
+Link board ETH is an expansion component for the :ref:`reel_board` and it needs
```

Comment:
I think this comment rather relates to reel_board doc.

Reply:
I have reworded it, reel board documentation is WIP.